style.css

```diff
@@ -6,28 +6,53 @@ body {
   margin: 0;
   padding: 0;
   font-family: 'Montserrat', sans-serif;
-  background: #0026ff;
 }
 
-h1 {
+
+.container{
+
+  position:relative;
+  width: 100%;
+  height: 100%;
+  font-size: 100px;
+ 
+
+}
+
+
+.container__video-background{
```

Comment:
Such a cool idea to add a video background and the choice you did really fits the subject of the travel bot.